Multisig/UI/App/UIKit/TransactionlListViewController/FlatTransactionsListViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  FlatTransactionListViewModel.swift
+//  Multisig
+//
+//  Created by Moaaz on 12/14/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct FlatTransactionsListViewModel {
+    var models: [SCG.TransactionSummaryItem] = []
```

Comment:
private?

Reply:
why ?

Reply:
because you have added convenience mutating functions. I think it it better to have one way to modify state

Reply:
But models var is used as well

Reply:
I see Andrey's point, I will make read only property.